training_config/decomposable_attention.json

```diff
@@ -11,15 +11,15 @@
       "end_tokens": ["@@NULL@@"]
     }
   },
-  "train_data_path": "/snli/snli_1.0_train.jsonl",
-  "validation_data_path": "/snli/snli_1.0_dev.jsonl",
+  "train_data_path": "https://s3-us-west-2.amazonaws.com/allennlp/datasets/te/snli_1.0_train.jsonl",
+  "validation_data_path": "https://s3-us-west-2.amazonaws.com/allennlp/datasets/te/snli_1.0_dev.jsonl",
   "model": {
     "type": "decomposable_attention",
     "text_field_embedder": {
       "tokens": {
         "type": "embedding",
         "projection_dim": 200,
-        "pretrained_file": "/glove/glove.6B.300d.txt.gz",
+        "pretrained_file": "https://s3-us-west-2.amazonaws.com/allennlp/datasets/glove/glove.6B.100d.txt.gz",
```

Comment:
This is the line that caused your problem.  You need 300d vectors here.  Sorry I missed this on the first pass.

Reply:
Oops, thank you!